doc/input-output-reference/src/overview/group-advanced-surface-concepts.tex

```diff
@@ -523,7 +523,8 @@ \subsection{Foundation:Kiva}\label{foundationkiva}
   However, multiple floors may exist in the same thermal zone so long as
   they reference separate Foundation:Kiva objects.
 \item
-  All foundation wall surfaces must be quadrilateral.
+  Foundation wall surfaces that are not triangular or quadrilateral may translate
+  well into the two-dimensional finite difference context.
```

Comment:
Did you mean "may *not* translate well . . ."?

Reply:
Yes. I did.